README.md

```diff
@@ -565,6 +566,18 @@ Platforms: all
 
 Note: For Android MediaPlayer, rate is only supported on Android 6.0 and higher devices.
 
+#### pitchAlgorithm
+Pitch algorithm for AVPlayerItem. [Apple documentation](https://developer.apple.com/documentation/avfoundation/avaudiotimepitchalgorithm).
+
+By default on iOS pitch algorithm is **lowQualityZeroLatency** which has limitations in allowed playback speed.
+
+* **lowQualityZeroLatency** - A low quality and very low computationally intensive pitch algorithm. This algorithm is suitable for brief fast-forward and rewind effects as well as low quality voice. The rate is snapped to {0.5, 0.666667, 0.8, 1.0, 1.25, 1.5, 2.0}.
```

Comment:
Add `(default)` next to this

Reply:
@kesha-antonov can you update the docs for this and then we can approve your PR

Reply:
@ashnfb @cobarx done

Reply:
@cobarx @ashnfb Hey guys. Any news on this PR?